src/pages/signin/ValidateCodeForm/BaseValidateCodeForm.js

```diff
@@ -175,7 +176,12 @@ class BaseValidateCodeForm extends React.Component {
             formError: {},
         });
 
-        Session.signIn('', this.state.validateCode, this.state.twoFactorAuthCode);
+        const accountID = lodashGet(this.props, 'credentials.accountID', null);
```

Comment:
No need for a 3rd param to `accountID` here. If not provided, the value will be `undefined`, which is fine.

```suggestion
        const accountID = lodashGet(this.props, 'credentials.accountID');
```